packages/swingset-liveslots/src/liveslots.js

```diff
@@ -1424,7 +1431,7 @@ function build(
       Fail`buildRootObject() for vat ${forVatID} returned ${rootObject} with no interface`;
     // Need to load watched promises *after* buildRootObject() so that handler kindIDs
     // have a chance to be reassociated with their handlers.
-    watchedPromiseManager.loadWatchedPromiseTable();
+    watchedPromiseManager.loadWatchedPromiseTable(unmeteredRevivePromise);
```

Comment:
This preserves existing behavior, but I would like to leave enough breadcrumbs to avoid such doubt in the future.

The first line of `revivePromise` asserts absence of metering, and then the rest proceeds to call `pRec = makePipelinablePromise(vpid)`, store the vpid–pRec mapping in `importedVPIDs`, register the vpid–promise mapping for the marshaller with `registerValue`, and return the promise.
`revivePromise` is called only by `loadWatchedPromiseTable`, which initializes an entry for the vpid–promise pair in `promiseRegistrations` and invokes `pseudoThen` to register settlement callbacks.

What commentary would you like to see at this invocation of `loadWatchedPromiseTable`?

Reply:
Good point, and yeah we don't have the time to dive any deeper than what you've just mapped out. Maybe just add "TODO: why does this need to be unmetered?", and we'll eventually do another pass through and revisit those decisions.